src/main/java/com/google/javascript/clutz/PlatformSymbols.java

```diff
@@ -1458,6 +1458,13 @@
           "TransitionEventInit",
           "Transport",
           "TreeWalker",
+          "TrustedHTML",
```

Comment:
Add some comments around these as they don't trully match the original comment at the top.

Reply:
Done.